earth_enterprise/BUILD_Ubuntu.md

```diff
@@ -22,7 +22,7 @@ Execute the following:
 
 ```bash
 curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | sudo bash
-sudo apt install git-lfs
+sudo apt-get install git-lfs
```

Comment:
minor: is there a reason for changing?

Reply:
I'm just making it consistent with the other lines. It just seemed odd to use apt here and apt-get everywhere else. Doesn't really make a difference.